crates/router/src/connector/powertranz/transformers.rs

```diff
@@ -74,11 +117,43 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for PowertranzPaymentsRequest
             )?,
             currency_code: diesel_models::enums::Currency::iso_4217(&item.request.currency)
                 .to_string(),
-            three_d_secure: false,
+            three_d_secure,
             source,
             order_identifier: item.payment_id.clone(),
             billing_address,
             shipping_address,
+            extended_data,
+        })
+    }
+}
+
+impl TryFrom<&types::PaymentsAuthorizeRouterData> for ExtendedData {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self, Self::Error> {
+        Ok(Self {
+            three_d_secure: ThreeDSecure {
+                challenge_window_size: 5,
```

Comment:
Why are we sending a hardcoded value for this field? Consider adding a comment in code if this is intentional.